fuse/eval/metrics/stat/metrics_stat_common.py

```diff
@@ -31,5 +34,118 @@ def __init__(
             target=target,
             mask=mask,
             metric_func=Stat.pearson_correlation,
-            **kwargs
+            **kwargs,
+        )
+
+
+class MetricSpearmanCorrelation(MetricDefault):
```

Comment:
I suggest that all of those metrics (including  MetricPearsonCorrelation) be put in a new folder for regression. Makes sense?

Reply:
yes makes a lot of sense, will refractor